app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventViewModel.java

```diff
@@ -50,11 +58,33 @@
     @NonNull
     public abstract Boolean canBeEdited();
 
+    public boolean openedAttributeList = false;
+
+    public void toggleAttributeList() {
+        this.openedAttributeList = !this.openedAttributeList;
+    }
+
     @NonNull
     public static ProgramEventViewModel create(@NonNull String uid, @NonNull String orgUnitUid, @NonNull String orgUnitName, @NonNull Date date,
                                                @NonNull State eventState, @NonNull List<Pair<String, String>> data, @NonNull EventStatus status,
                                                @NonNull Boolean isExpired, @NonNull String attributeOptionComboName, Geometry geometry, Boolean canBeEdited) {
         return new AutoValue_ProgramEventViewModel(uid, orgUnitUid, orgUnitName, date, eventState, data, status, isExpired, attributeOptionComboName, geometry, canBeEdited);
     }
 
+    public SpannableStringBuilder getSpannableStringValues() {
```

Comment:
can you add a more descriptive name? thanks ! getSpannableString it seems generic and you're getting specific strings